### ISSUE TITLE
Uncaught TypeError: Cannot read property 'save' of undefined

### ISSUE THREAD

**Issue by enroxorz** [moxley/atom-ruby-test#33]
Uncaught TypeError: Cannot read property 'save' of undefined

**Atom Version**: 0.158.0
**System**: Mac OS X 10.9.5
**Thrown From**: [ruby-test](https://github.com/moxley/atom-ruby-test) package, v0.9.1

### Steps To Reproduce
(On Mac)
1. Open a Cucumber based project
2. Go to Packages -> Ruby Test -> Run test file

Note: No file is open

### Stack Trace

```
At /Users/enri8294/.atom/packages/ruby-test/lib/ruby-test-view.coffee:61

TypeError: Cannot read property 'save' of undefined
  at RubyTestView.module.exports.RubyTestView.runTest (/Users/enri8294/.atom/packages/ruby-test/lib/ruby-test-view.coffee:61:37)
  at RubyTestView.module.exports.RubyTestView.testFile (/Users/enri8294/.atom/packages/ruby-test/lib/ruby-test-view.coffee:46:6)
  at atom-workspace.<anonymous> (/Users/enri8294/.atom/packages/ruby-test/lib/ruby-test-view.coffee:21:59)
  at atom-workspace.handler (/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:156:19)
  at Ipc.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:65:30)
  at Ipc.emit (events.js:107:17)
  at process.<anonymous> (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/ipc.js:22:29)
  at process.emit (events.js:110:17)

```

/cc @atom/core

**Comment by eleidan** [comment]
1. Open a project with *RSpec* tests.
2. Press `ctrl-shift-Y`

**Atom Version**: 0.176.0
**System**: Ubuntu 14.04, linux 3.13.0-44-generic
**Thrown From**: [ruby-test](https://github.com/moxley/atom-ruby-test) package, v0.9.4

### Stack Trace

Uncaught TypeError: Cannot read property 'save' of undefined

```
At /home/<user_name>/.atom/packages/ruby-test/lib/ruby-test-view.coffee:61

TypeError: Cannot read property 'save' of undefined
  at RubyTestView.module.exports.RubyTestView.runTest (/home/<user_name>/.atom/packages/ruby-test/lib/ruby-test-view.coffee:61:37)
  at RubyTestView.module.exports.RubyTestView.testAll (/home/<user_name>/.atom/packages/ruby-test/lib/ruby-test-view.coffee:52:6)
  at atom-workspace.<anonymous> (/home/<user_name>/.atom/packages/ruby-test/lib/ruby-test-view.coffee:24:58)
  at atom-workspace.handler (/usr/share/atom/resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/usr/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/usr/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:170:20)

```

### Commands

```
     -0:00.7 ruby-test:test-all (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "symbols-view"
    ],
    "themes": [
      "seti-ui",
      "monokai"
    ]
  },
  "ruby-test": {}
}
```

### Installed Packages

```coffee
# User
atom-ctags, v2.5.5
autocomplete-plus, v2.0.2
autocomplete-snippets, v1.0.0
git-log, v0.2.0
linter, v0.10.1
linter-bootlint, v0.0.3
linter-coffeelint, v0.1.7
linter-csslint, v0.0.11
linter-erb, v0.0.3
linter-less, v0.3.1
linter-scss-lint, v0.0.11
linter-shellcheck, v0.0.6
monokai, v0.12.0
project-manager, v1.15.4
ruby-test, v0.9.4
seti-ui, v0.6.3

# Dev
No dev packages
```

/cc @atom/core

**Comment by moxley** [comment]
Fixed in v0.9.5. Thanks!
